tools/Microsoft.AspNetCore.Server.Kestrel.GeneratedCode/run.proj

```diff
@@ -0,0 +1,10 @@
+<Project ToolsVersion="15.0">
```

Comment:
Seems like this file adds a layer we don't need. makefile.shade can just call dotnet run on the project directly with the arguments, right?

Also, to make 'dotnet run' just do the right thing by default, you can add this M.A.S.GenerateCode.csproj
```xml
<PropertyGroup>
   <StartArguments>
    &quot;$(MSBuildProjectDirectory)/../../src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/FrameHeaders.Generated.cs&quot;
    &quot;$(MSBuildProjectDirectory)/../../src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/Frame.Generated.cs&quot;
  </StartArguments>
</PropertyGroup>
```